bin/MessagesBundle_es.properties

```diff
@@ -114,6 +114,22 @@ extFCJ_c=Activar extensión de FidoCadJ mientras se copian elementos
 Restart=Reiniciar
 Drawing=Dibujar
 FidoCad=Extensiones FidoCadJ
+
+Theme_management=Theme and color management
```

Comment:
Put all the untranslated resources at the end of the file. They are easier to find for those who would like to work on them.